modules/listeners.py

```diff
@@ -0,0 +1,39 @@
+import discord
+from discord.ext import commands
+import traceback
+import sys
+
+
+class Listeners(commands.Cog):
```

Comment:
I like this, but I'll have to implement Cog visibilities in the `Status` module before I can approve this module.